build/build.sh

```diff
@@ -82,9 +83,14 @@ if [ "${BUILD_PDF:-}" != "false" ] && [ -z "$DOCKER_EXISTS" ]; then
   rm images
 fi
 
-# Create PDF output (unless BUILD_PDF environment variable equals "false")
+# If Docker is available, use athenapdf to create PDF
 if [ "${BUILD_PDF:-}" != "false" ] && [ -n "$DOCKER_EXISTS" ]; then
   echo >&2 "Exporting PDF manuscript using Docker + Athena"
+  if [ "${CI:-}" = "true" ]; then
+    # Incease --delay for CI builds to ensure the webpage fully renders, even when the CI server is under high load.
+    # Local builds default to a shorter --delay to minimize runtime, assuming proper rendering is less crucial.
+    echo >&2 "Continuous integration build detected. Setting athenapdf --delay=${MANUBOT_ATHENAPDF_DELAY:=5000}"
```

Comment:
Setting the default value of a variable as a side-effect of printing a log message does not lead to understandable code.

Reply:
Not sure I totally prefer https://github.com/manubot/rootstock/pull/255/commits/44576670255d8a8482bdc6bdddb120a898396bce, but if it will be more widely understood, then OKAY with me.